etna/transforms/math/limit.py

```diff
@@ -0,0 +1,122 @@
+from typing import List
+import numpy as np
+import pandas as pd
+
+from etna.datasets import TSDataset
+from etna.datasets import set_columns_wide
+from etna.transforms.base import ReversibleTransform
+from etna.transforms.utils import match_target_quantiles
+
+
+class LimitTransform(ReversibleTransform):
+    """LimitTransform limits values of some feature between the borders"""
+
+    def __init__(self, in_column: str, lower_bound: float = -1e10, upper_bound: float = 1e10):
+        """
+        Init LimitTransform.
+
+        Parameters
+        ----------
+        in_column:
+            column to apply transform
+        lower_bound:
+            lower bound for the value of the column
+        upper_bound:
+            upper bound for the value of the column
+        """
+        super().__init__(required_features=[in_column])
+        self.in_column = in_column
+        self.lower_bound = lower_bound
+        self.upper_bound = upper_bound
+        self.tol = 1e-10
+
+    def _get_column_name(self) -> str:
+        return self.in_column
+
+    def _fit(self, df: pd.DataFrame) -> "LimitTransform":
+        """Fit method does nothing and is kept for compatibility.
+
+        Parameters
+        ----------
+        df:
+            dataframe with data.
+
+        Returns
+        -------
+        result: AddConstTransform
+        """
+        return self
+
+    def fit(self, ts: TSDataset) -> "LimitTransform":
+        """Fit the transform."""
+        super().fit(ts)
+        return self
+
+    def _transform(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Apply scaled logit transform to the dataset.
+
+        Parameters
+        ----------
+        df:
+            dataframe with data to transform.
+
+        Returns
+        -------
+        result: pd.Dataframe
+            transformed dataframe
+        """
+        result = df
+        features = df.loc[:, pd.IndexSlice[:, self.in_column]]
```

Comment:
`df` already contains only `self.in_column` [see](https://github.com/etna-team/etna/blob/1d20ffc07f90e76cc9fb0717a6e9887551ebd442/etna/transforms/base.py#L126) -- some Transforms might contain such slices, because in the previous major version we had another contract